splight_lib/models/base.py

```diff
@@ -38,8 +38,7 @@ def save(self):
         files_dict = self._get_model_files_dict()
         saved = self._db_client.save(
             self.__class__.__name__,
-            # TODO: improve the following line
-            json.loads(self.model_dump_json(exclude_none=True)),
+            self.model_dump(exclude_none=True, mode="json"),
```

Comment:
According to this issue https://github.com/pydantic/pydantic/issues/1409#issuecomment-1423995424 this is the way to solve this

Reply:
I was thinking of using a custom serializer for the datetime field, that may involve some nasty code for the auto-generated models like routines. Let's try this approach